### DISCUSSION TITLE
Localization 🌍

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#208] [Ideas]
As some users don't speak english, it could be interesting to offer the software in other languages.

This is probably not something we want to prioritize in the short-term though since it could heavily slow the development workflow. 
One alternative we could consider to do this without slowing the development workflows is to have translations be auto-generated through CI, however this would lead to lower quality translations since automatic translation can't perfectly capture nuances and context.

**Comment by ElliottCuvelier** [comment]
If you plan to localize, don't forget custom objects and custom fields. This might be a problem in multi-country companies, since English is not widely used, particularly in sales teams.

**Reply by FelixMalfait** [reply]
Good point! Thanks

**Comment by richardwiden** [comment]
Being able to choose formats should probably be implemented well before actual localization. US-styled dates and calendars are especially annoying to deal with for us iso-standard people.

**Comment by dntrply** [comment]
+1 Would love support for Indian languages.

Is it so bad if CI auto generates translations? Once the initial translations (accompanied by support for localization) are in place, maybe the community will identify nuance and help improve the translations?

**Comment by FelixMalfait** [comment]
We did prioritize localization which is live now! We don't support Indian languages through sorry :(. We may add more in the future!
